xclim/sdba/base.py

```diff
@@ -50,6 +50,7 @@ def __init__(
         window: int = 1,
         add_dims: Optional[Sequence[str]] = None,
         interp: Union[bool, str] = False,
+        manage_chunking: bool = True,
```

Comment:
This publicly exposes something that feels like an implementation detail. Does this need to be part of the public API ? 

There is a tradeoff between flexibility and accessibility. Each time we add another argument, we implicitly ask users  to understand what it does, and then decide which values they should use. At the end, it makes the software less accessible.

Reply:
You are right, this one could go. I used it to test different scenarios, but it turns out that managing chunking myself only save a few seconds, nothing significative enough to merit a public option. I think there still could be use cases where optimal performance is obtained when `group.apply` does not rechunk its output, but am willing to sacrifice it if I don't find a better option.